core/src/main/java/co/nimblehq/compose/crypto/core/BaseViewModel.kt

```diff
@@ -9,6 +9,8 @@ import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.flow.*
 import kotlinx.coroutines.launch
 
+const val FIAT_CURRENCY = "usd"
```

Comment:
Shall we create new constant class? IMO it is better to have a separate constant class rather than having this inside the `BaseViewModel` class.

Reply:
@kaungkhantsoe I thought to it already. But after reviewing the code implementation, it was put in the `ViewModel` before. We only use it for `ViewModel`, hence, putting it in the `BaseViewModel` does make sense too 🙏

Reply:
@doannimble No, we are gonna use it inside test classes as well. I am currently implementing it.

Reply:
@kaungkhantsoe Okay, I will move it to `core`, inside a constant file